back/app/models/que_job.rb

```diff
@@ -1,5 +1,30 @@
 # frozen_string_literal: true
 
+# == Schema Information
+#
+# Table name: public.que_jobs
+#
+#  priority             :integer          default(100), not null
+#  run_at               :timestamptz      not null
+#  id                   :bigint           not null, primary key
+#  job_class            :text             not null
+#  error_count          :integer          default(0), not null
+#  last_error_message   :text
+#  queue                :text             default("default"), not null
+#  last_error_backtrace :text
+#  finished_at          :timestamptz
+#  expired_at           :timestamptz
+#  args                 :jsonb            not null
+#  data                 :jsonb            not null
+#  job_schema_version   :integer          default(1)
+#
+# Indexes
+#
+#  que_jobs_args_gin_idx                 (args) USING gin
+#  que_jobs_data_gin_idx                 (data) USING gin
+#  que_poll_idx                          (queue,priority,run_at,id) WHERE ((finished_at IS NULL) AND (expired_at IS NULL))
+#  que_poll_idx_with_job_schema_version  (job_schema_version,queue,priority,run_at,id) WHERE ((finished_at IS NULL) AND (expired_at IS NULL))
+#
```

Comment:
Seems this auto-generated comments section hadn't been generated, and my migration in this PR generated it.

Reply:
No explanation. I have this all the time. It seems that the library we use for annotation does not always pick up the tables right away, but I didn't investigate it further. After all, those are just comments ;)

Reply:
What gem does the annotating? I see we had some gem updates yesterday ([update to Rails version](https://github.com/CitizenLabDotCo/citizenlab/pull/7104/files))